tensorboard/webapp/plugins/plugins_container_test.ts

```diff
@@ -231,6 +239,7 @@ describe('plugins_component', () => {
     });
 
     it('creates an element for IFRAME type of plugin', async () => {
+      const pluginApiHost = TestBed.inject(PluginApiHostModule);
```

Comment:
Hmm how does this actually work? I see no providers for this.

Reply:
The beforeEach calls setup(). setup(), by default, injects the PluginApiHostModule. Only in the new test do we choose not to inject PluginApiHostModule.

Reply:
Ah, `getTestingProvider` includes the PluginApiHostModule. Make sense.

Reply:
Ah, right, ya, unfortunately that's hidden by a layer of indirection.